test-requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/OCA/product-attribute@refs/pull/1389/head#subdirectory=setup/product_origin
```

Comment:
Hi @sbidoul. 

Question : once setup folder has been removed, replaced by toml file, what is the correct syntax to write such dependency in the futur ? 

thanks !

Reply:
Thanks ! very clear.